<launch>
  <!-- Joystick device path.
  Default is /dev/js1. If an environment variable JOYSTICK_DEVICE
  is set then this replaces the default. If the launch file is started with
  the argument js:=blah then blah overides both the default and the environment. -->
  <arg name="js" default="$(optenv JOYSTICK_DEVICE /dev/js1)" />

  <!-- Joystick driver node -->
  <node pkg="joy" type="joy_node" name="joystick">
	<param name="dev" value="$(arg js)" type="string"/>
	<param name="autorepeat_rate" value="10.0" type="double"/>
  </node>

  <!-- Base teleop -->
  <node pkg="scitos_teleop" type="rumble_control" name="teleop_joystick" output="screen">
	<param name="scale_angular" value=".7" type="double"/>
	<param name="scale_linear" value=".7" type="double"/>
  </node>

  <!-- Head teleop -->
  <node pkg="scitos_teleop" type="head_joy.py" name="teleop_joystick_head_control" />
  <node pkg="scitos_teleop" type="eye_joy.py" name="teleop_joystick_eye_control" />
</launch>
